### PR TITLE
add url validation to commit model

### PR DESCRIPTION
**Pull request by lalithr95**
URL validation is done through URI regex and necessary tests are also listed

### CODE REVIEW

File: test/models/commit_test.rb [hunk 1]
```diff
@@ -25,4 +25,21 @@ class CommitTest < ActiveSupport::TestCase
     commit = create(:commit, sha1: '1234567')
     assert_includes commit.version, '12345'
   end
+
+  test "validates valid URL" do
+    valid_url = ['https://github.com/ruby-bench/ruby-bench-web/blob/master/app/models/commit.rb',
```
Comment (tgxworld) [inline test/models/commit_test.rb]: Nitpick here: I prefer 
```
valid_url = [
  'https://...',
  'http://...'
]
```

:smile:

File: test/models/commit_test.rb [hunk 2]
```diff
@@ -25,4 +25,21 @@ class CommitTest < ActiveSupport::TestCase
...
+      'http://github.com/ruby-bench/ruby-bench-web/blob/master/app/models/commit.rb']
```
Comment (tgxworld) [inline test/models/commit_test.rb]: Nitpick: Add a newline here :smile:

**Comment by tgxworld** [comment]
PR looks good to me :) :+1:

**Comment by lalithr95** [comment]
@tgxworld updated changes :smiley:

**Comment by tgxworld** [comment]
:+1:

**Pull request merged**